src/main/java/net/openhft/chronicle/bytes/BytesUtil.java

```diff
@@ -131,6 +131,7 @@ static int[] isTriviallyCopyable0(@NotNull Class<?> clazz) {
      * @param fields The fields of the class.
      * @return An array of two integers, where the first element is the minimum offset, and the second element is the maximum offset.
      */
+    @SuppressWarnings("deprecation")
```

Comment:
We shouldn't be suppressing deprecation warnings, that will make these things very hard to find down the track. Unless the method itself is due to go at the same time as the deprecated thing it references.

Reply:
I have narrowed the SuppressWarning tot he field that needs it